src/live.asp.net/Controllers/AdminController.cs

```diff
@@ -89,12 +93,9 @@ public class AdminInputModel
                 input.LiveShowEmbedUrl = "https://" + input.LiveShowEmbedUrl.Substring("http://".Length);
             }
 
-            TrackShowEvent(input, liveShowDetails);
+            TrackShowEvent(input, input);
```

Comment:
Second one is `ILiveShowDetails` don't think the logic in `TrackShowEvent` works for the new approach though 😢